NewForce-Cohort-4/nutshell-team-odd-eclipse
# Dtb events scroll and highlight

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Copy content from API.txt file in /nuttshell-team-odd-eclipse into /api/database.json (only do this if you need more data to test features)
- Refresh site so that login and register appear
- Login as me@me.com. Check to see if the appropriate events are listed, and if there are more than one event for that user, ensure that they are listed chronologically and that the top event is emphasized compared to the other events.
- Check that the scrolling functionality works.
- Logout and ensure that the DOM is cleared
- Login as you@you.com and check functionality the same as above with me@me.com.
- Delete one of the events to check the functionality of the delete button. Make sure that the list refreshes.
- 
# Checklist:
